app/views/application/_auth_providers.html.erb

```diff
@@ -12,34 +12,10 @@
 
     <div class="col justify-content-center d-flex align-items-center flex-wrap gap-2">
       <% Auth.providers.each do |provider| %>
-        <% if provider == "openid" %>
-          <%= button_tag image_tag("auth_providers/openid.svg",
-                                   :alt => t(".openid.alt"),
-                                   :size => "36"),
-                         :type => "button",
-                         :data => { "bs-toggle" => "collapse",
-                                    "bs-target" => "#login_auth_buttons, #openid_login_form" },
-                         :title => t(".openid.title"),
-                         :class => "btn btn-outline-secondary border p-2" %>
-        <% elsif provider != @preferred_auth_provider %>
+        <% if provider != @preferred_auth_provider %>
           <%= auth_button provider %>
-        <% end -%>
+        <% end %>
       <% end -%>
     </div>
   <% end %>
-
-  <%# :tabindex starts high to allow rendering at the bottom of the template %>
-  <%= form_tag auth_path(:provider => "openid"), :id => "openid_login_form", :class => "collapse" do %>
-    <div class="mb-3">
-      <label for="openid_url" class="form-label">
-        <%= image_tag "auth_providers/openid.svg", :size => "36", :alt => "", :class => "align-text-bottom" %>
-        <%= t ".openid_url" %>
-      </label>
-      <%= hidden_field_tag("referer", params[:referer], :autocomplete => "off") %>
```

Comment:
Ok, I think I know what's going on. We used to have the same "referer" field twice on the /user/new page, once on the openid form (this line), and another time as field  in the "new_user" form:   `<%= hidden_field_tag("referer", h(@referer)) unless @referer.nil? %>` (new.html.erb).

Since the field in the openid form was always present, auth_providers.js never returned "undefined". Now that we're removing this form, this assumption no longer holds true.

So the change in auth_providers.js to have an empty string as fallback should be backwards compatible. Previously, the "auth buttons" always had a referer URL parameter, even if the value was an empty string.

Reply:
I think that's an existing bug to be honest but preserving the status quo is fine for this pull request.